pom.xml

```diff
@@ -164,6 +165,12 @@
         <artifactId>junit</artifactId>
         <version>4.12</version>
         <scope>test</scope>
+        <exclusions>
```

Comment:
hmmm...I don't think this is necessary is it?

Reply:
Removing this results in warnings being generated in the other modules because of the usage of `all` instead of `core` in some modules. Also the version number needs to be explicitly mentioned `${hamcrest.version}` at the `pom` of each of these modules to get this to build but `ildcp-core` strangely starts to give an issue when I do this. Leaving this as is for now.

Reply:
In the branch I'm working on, I only use `hamcrest-core`. I had to change some imports in a few projects, but it should work here too.